src/components/CromwellModal.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonPrimary, spinnerOverlay } from 'src/components/common'
+import { withModalDrawer } from 'src/components/ModalDrawer'
+import { tools } from 'src/components/notebook-utils'
+import TitleBar from 'src/components/TitleBar'
+import { Ajax } from 'src/libs/ajax'
+import { withErrorReporting } from 'src/libs/error'
```

Comment:
Use `withErrorReportingInModal` to get this modal to properly dismiss when errors occur, just added in a PR recently :)

Reply:
@jdcanas I need more information about how `withErrorReportingInModal` works. When I tried switching to it, the modal would remain open when I hit an error, but I would not see the error itself. And when I closed the modal, the popup that used to show the error message was not present.

What do I need to do to full implement `withErrorReportingInModal`?

Reply:
@jdcanas Passing in onDismiss did fix the issue. I have pushed up the change.